explorer/frontend/scripts/regen-shell.nix

```diff
@@ -0,0 +1,14 @@
+# args@{...}: (import ./default.nix (args // { cardano-sl-explorer = null; })).shell
+
+{ pkgs ? import <nixpkgs> {} }:
```

Comment:
should probably use `locallib.fetchNixPkgs` here, or the version will be in flux and can fail

Reply:
Fixed